tests/xfstests/run.pm

```diff
@@ -254,6 +254,12 @@ sub run {
         if ($type eq $HB_DONE) {
             # Test finished without crashing SUT
             log_add($STATUS_LOG, $test, $status, $time);
+            if ($status =~ /FAILED/) {
+                my $cmd = "cat /opt/xfstests/results/$category/$num.out.bad | tee $LOG_DIR/$category/$num.out.bad";
```

Comment:
just out of curiosity what the benefits ( if any ) cat+tee have over "regular" cp. because what you are doing here you simply copying a file?

Reply:
@asmorodskyi Just in case the log missing by some reason(LOG_DIR in /tmp), and at least I could find log in snapshot. I can find a better place for log, except /tmp in next PR. It really has some problem when kdump happened, and reboot will remove some useful information.